modules/behavioural_qc/php/models/incompletedto.class.inc

```diff
@@ -19,6 +19,83 @@ namespace LORIS\behavioural_qc\Models;
  */
 class IncompleteDTO implements \LORIS\Data\DataInstance
 {
+    /**
+     * The instrument
+     *
+     * @var string
+     */
+    public $instrument = '';
```

Comment:
private

Reply:
@xlecours that will make all the variables require to have underscores because of phan. Then the json values on the frontend will all be underscored as well. Anyway to avoid that?

Reply:
@maltheism add the underscore in the aliases of the query in the provisioner.